examples/sdkconfig.defaults

```diff
@@ -8,6 +8,7 @@ CONFIG_FREERTOS_HZ=1000
 CONFIG_ESP32S2_DEFAULT_CPU_FREQ_240=y
 CONFIG_ESP32S3_DEFAULT_CPU_FREQ_240=y
 CONFIG_ESP_SYSTEM_EVENT_TASK_STACK_SIZE=4096
+CONFIG_PTHREAD_TASK_STACK_SIZE_DEFAULT=4096
```

Comment:
A task is created to manage the timers we spawn, it would often time exceed it's stack limit and overwrite which would potentially lock the esp32 up.